src/applications/personalization/profile/components/alerts/LoadFail.jsx

```diff
@@ -26,10 +25,6 @@ export default function LoadFail() {
   );
 }
 
-LoadFail.propTypes = {
-  information: PropTypes.string.isRequired,
-};
-
```

Comment:
This is just cleanup that was noticed, the information prop is no longer used in this component at all.